pages/governance/proposing_a_new_market.md

```diff
@@ -0,0 +1,94 @@
+# Proposing a new market
+
+## Example Proposal Json
+Below is an example proposal JSON file to propose adding `BTC-USD` as a new perpetual market (if it had not been added yet).
+
+```
+{
+    "title": "Add BTC-USD perpetual market",
+    "deposit": "10000000000000000000000adv4tnt",
```

Comment:
is this fixed or a recommendation?

Reply:
The required deposit, so I think the explanation is out-of-scope for this documentation. As an initial version for this doc I think it's fair to have sponsors figure this out (it's pretty straightforward for anyone who has made a proposal)

Reply:
Makes sense

Reply:
Should we reference the calculator anywhere that Yujin put together a while back? Think that could be helpful (if its still accurate) https://docs.google.com/spreadsheets/d/1DDOBjg3UPehSKiaw9g4l4NKNEuKd26Os6LtRe6gKlE0/edit#gid=0

Reply:
> Should we reference the calculator anywhere that Yujin put together a while back?

We discussed this [here](https://www.notion.so/Proposing-a-new-market-4042cb25cef34707baf7058f19830f77?d=bf30c27299bd4c1a858c769bbc4df7fc&pvs=4#a9e8827897804222b6a8e91618de6c91).

Reply:
ok, I personally think it's helpful to publish it (if its accurate). I find this file a bit confusing to figure out what the formulas are to get to the right outputs for some of these params